manifests/vhost.pp

```diff
@@ -397,6 +405,21 @@
     }
   }
 
+  if $modsec_audit_log_file {
```

Comment:
@sathieu This logic is confusing. You should always check the "manage" boolean first and determine path variables after. Something like:

```
if $mod_audit_log {
  if $modsec_audit_log_file {
    $modsec_audit_log_destination = "${logroot}/${modsec_audit_log_file}"
  } else if $modsec_audit_log_pipe { 
    $modsec_audit_log_destination = $modsec_audit_log_pipe
  } else {
    if $ssl {
       $modsec_audit_log_destination = "${logroot}/${name}_security_ssl.log"
     } else {
       $modsec_audit_log_destination = "${logroot}/${name}_security.log"
     }
} else {
    $modsec_audit_log_destination = undef
}
```

This way you only set variables/paths if `$modsec_audit_log` is set to true.